experimental/babel-preset-env/CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ And add them in this structure:
 
 ```js
 // es2016
-"@babel/transform-exponentiation-operator": {
```

Comment:
actually this whole document is kind of out of date

https://github.com/hulkish/babel/blob/master/experimental/babel-preset-env/data/plugin-features.js, referencing the old way to do things + wrong links

Reply:
This link is fixed 👍